plugin/extraction_plugin.py

```diff
@@ -10,13 +10,14 @@ class Plugin(ExtractionPlugin):
     def plugin_info(self):
         plugin_info = PluginInfo(
             self,
-            name='your_plugin_name',
             version='0.0.0',
             description='description of your plugin',
             author=Author('Your name', 'your@email.address', 'your organisation'),
             maturity=MaturityLevel.PROOF_OF_CONCEPT,
             webpage_url='',  # e.g. url to the code repository of your plugin
-            matcher='* and $data.type=raw'
+            matcher='$data.type=raw',
```

Comment:
Maybe worth a line comment that people can write their query here? And this fixes the bug that `and` was in lowercase, so that's nice. Lowercase and becomes a search term right, instead of an operator?

```suggestion
            matcher='$data.type=raw',  # add the query for the types of traces your plugin should match
```